style.css

```diff
@@ -156,6 +161,8 @@ h1, h2{
 	#cheat-sheet pre{
 		margin-top: 0;
 		margin-bottom: 5px;
+		/* works better on my printer, need to test more */
+		/* background: #dddddd; */
```

Comment:
Can these commented lines be left out for now? At least until you've tested them.

Reply:
Sure! But I dont seem to find the option to add this change to this existing PR... I've pushed it into my forked master. If I do the new pull request it sends me over to "view pull request" (the existing one) and I see my latest commit "removed comments). But all the way down it says "No commit comments for this range"

Reply:
Another question: check this commit https://github.com/limzykenneth/p5-cheat-sheet-html/commit/951e84f7835084df4bc6efd44b09f5f16f6663e9 

I've only fixed this in my own nl-main.js, not the other ones. Should I update them as well to keep it clean?

Reply:
When you push to the branch you compared with (master in this case) the PR will automatically update so no need for further actions.

i didn't notice commit https://github.com/limzykenneth/p5-cheat-sheet-html/pull/1/commits/951e84f7835084df4bc6efd44b09f5f16f6663e9 can you explain why that is changed? Especially for the text labels, they were labeled that way to be in line with the original cheat sheet.

Reply:
I have pushed to my master... see this commit https://github.com/vincentsijben/p5-cheat-sheet-html/commit/b9d90a707f34d8eb3edee7cdbf0c52e67f464b8f (but it is not showing up in the PR)

I have removed the "1" for x and y. x1 should only be used if there is a x2 :) That's also 'wrong' in the original PDF.
In an ellipse, x1 implies there would be an x2... it should be ellipse(x,y,width,height) not ellipse(x1,y1,width,height)

Reply:
ah, it took some time I guess, it showed up: https://github.com/limzykenneth/p5-cheat-sheet-html/pull/1/commits/b9d90a707f34d8eb3edee7cdbf0c52e67f464b8f

So have a look at my remark about the x1. Then you could merge.

Reply:
I'm not aiming for any opinion in terms of preference for `x1` or `x` here, I would suggest that you can file an issue or PR over at @bmoren's repo about this after I have move this cheat sheet over there. Can you revert those changes for now?

Reply:
ok I have pushed a reverted commit. All should be ok now. I'll file a PR for @bmoren